### PR TITLE
Update Auth CODEOWNERS

### PR DESCRIPTION
**Pull request by prameshj**
@sam-gc @lisajian @yuchenshi @avolkovi PTAL.

Thanks!

### CODE REVIEW

File: .github/CODEOWNERS
Comment (lisajian) [inline .github/CODEOWNERS:64]: Is @renkelvin the github username that you meant to include instead of chuanr?

Comment (prameshj) [reply .github/CODEOWNERS]: Yes, updated :) Thanks!

**Review by lisajian — APPROVED** [review]
LGTM, pending approval from the others so they're aware of the change too - thanks Pavithra!

**Review by sam-gc — APPROVED** [review]

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**
